cmd/ci-operator-prowgen/main.go

```diff
@@ -599,6 +599,9 @@ func generateJobs(
 			}
 		} else {
 			periodics = append(periodics, *generatePeriodicForTest(element.As, info, label, podSpec, true, *element.Cron, configSpec.CanonicalGoRepository))
+			dupe := *generatePeriodicForTest(element.As+"-build01", info, label, podSpec, true, *element.Cron, configSpec.CanonicalGoRepository)
```

Comment:
@stevekuznetsov 

Nothing added if `if element.ContainerTestConfiguration != nil ` like we do for presubmits.
So i removed it.